stable/fairwinds-insights/ci/test-values.yaml

```diff
@@ -0,0 +1,52 @@
+image:
+  tag: 3.2.4
+
+ingress:
+  hostedZones:
+  - test.cluster.local
+
+options:
+  insightsSAASHost: "https://staging.insights.fairwinds.com"
```

Comment:
remove `staging`?

Reply:
This is the file for CI and I was hesitant to hit prod and build up that data. I don't know if that's overly paranoid though.

Reply:
seems like a good idea to me.